cpp/src/aztec/honk/composer/standard_honk_composer.test.cpp

```diff
@@ -0,0 +1,86 @@
+#include "standard_honk_composer.hpp"
+#include <gtest/gtest.h>
+
+TEST(standard_honk_composer, test_sigma_correctness)
```

Comment:
Are all the components in place for us to add a test of `assert_equal` as well?

Reply:
Did we have a test like this somewhere in legacy?

Reply:
Maybe not? I think we probably should, but it's also true that `assert_equal` will be tested elsewhere (e.g., we should test verifier fails when an untrue `assert_equal` is made), so it's not necessary to add anything. I was thinking we could add a test illustrating how it works under the hood.